README.md

```diff
@@ -32,36 +32,51 @@ cd kagome
 
 mkdir build && cd build
 cmake -DCMAKE_BUILD_TYPE=Release ..
-make kagome -j 
-
+make kagome -j $(( $(nproc 2>/dev/null || sysctl -n hw.ncpu) + 1 ))
 ```
+
+`make kagome -j <numbers of cores>` (should be less than you have, here it is half of your cores + 1)
```

Comment:
Hm, is it a half of cores, though? It seems like all cores. Maybe you meant that the number of virtual cores is doubled when hypertheading is enabled, but it's kind of a specific case.